configs/AXISARGUSF745AIO/config.h

```diff
@@ -23,7 +23,7 @@
 
 #define FC_TARGET_MCU     STM32F7X2
 
-#define BOARD_NAME        ARGUSF7AIO
+#define BOARD_NAME        AXIS_ARGUS_F745_AIO
```

Comment:
![image](https://github.com/user-attachments/assets/4b44d2c0-1443-4db0-98f6-1f4595342eb1)

Perhaps

```suggestion
#define BOARD_NAME        AXISFLYINGF745AIO
```
```suggestion
#define BOARD_NAME        AXISARGUSF745AIO
```

Note: we also have https://github.com/betaflight/config/pull/581

Reply:
i agree. if agreed by @sugaarK , commit, i will approve.